Header header
geometry_msgs/Point32[] hull_vertices
float64 slope